nexus/types/src/external_api/views.rs

```diff
@@ -390,80 +390,13 @@ pub struct IpPool {
     pub identity: IdentityMetadata,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
-pub struct Ipv4Utilization {
-    /// The number of IPv4 addresses allocated from this pool
-    pub allocated: u32,
-    /// The total number of IPv4 addresses in the pool, i.e., the sum of the
-    /// lengths of the IPv4 ranges. Unlike IPv6 capacity, can be a 32-bit
-    /// integer because there are only 2^32 IPv4 addresses.
-    pub capacity: u32,
-}
-
-#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
-pub struct Ipv6Utilization {
-    /// The number of IPv6 addresses allocated from this pool. A 128-bit integer
-    /// string to match the capacity field.
-    #[serde(with = "U128String")]
-    pub allocated: u128,
-
-    /// The total number of IPv6 addresses in the pool, i.e., the sum of the
-    /// lengths of the IPv6 ranges. An IPv6 range can contain up to 2^128
-    /// addresses, so we represent this value in JSON as a numeric string with a
-    /// custom "uint128" format.
-    #[serde(with = "U128String")]
-    pub capacity: u128,
-}
-
+/// The utilization of IP addresses in a pool.
```

Comment:
```suggestion
/// The utilization of IP addresses in a pool.
///
/// Note that both `capacity` and `remaining` are integer values represented as double-precision floating-point values to accommodate allocations larger than a 64-bit integer (which is common when using IPv6 addresses). With very large pools (> 2^53) integer precision may be lost, but in such a case the pool has many (many) available addresses.
```

I think it bears some small amount of explanation.

Reply:
That's very helpful, thanks.

Reply:
Fleshed this out in 45419e0ba